Cargo.toml

```
@@ -15,6 +15,7 @@ members = [
     "single_offer_router",
     "events",
     "timelock",
+    "token_contract"
```

I don't think the `_contract` suffix is necessary. We don't use it for others for their directory name.
```suggestion
    "token"
```

---

Done